packages/in_app_purchase/README.md

```diff
@@ -26,14 +26,14 @@ guides:
 * [In-App Purchase (App Store)](https://developer.apple.com/in-app-purchase/)
 * [Google Play Biling Overview](https://developer.android.com/google/play/billing/billing_overview)
 
-You can check out the [example app README](example/README.md) for steps on how
+You can check out the [example app README](https://github.com/flutter/plugins/blob/master/packages/in_app_purchase/example/README.md) for steps on how
```

Comment:
I remember @mklim mentioned that we couldn't use the absolute path here for some reason, maybe I remembered incorrectly.

Reply:
For me, the reason was that the links do not work on Pub. The absolute paths work from anywhere.

Reply:
I think I was saying that they're slightly less convenient than working relative links in that we'll need to update them more if these files move around at all, but they should work fine. It is better to fix them than leave them broken.